hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/CryptoService.java

```diff
@@ -46,7 +43,9 @@ public interface CryptoService extends Service {
     @Override
     @NonNull
     CryptoPreTransactionHandler createPreTransactionHandler(
-            @NonNull States states, @NonNull PreHandleContext ctx);
+            @NonNull States states,
+            @NonNull PreHandleContext ctx,
+            @NonNull PreHandleTxnAccessor accessor);
```

Comment:
I am probably missing something, but I think we have a chicken and egg problem here. To call this method, we need an instance of `PreHandleTxnAccessor`, which provides access to a `CryptoPreTransactionHandler`. But how can this be created before we call the method `CryptoService.createPreTransactionHandler()`?